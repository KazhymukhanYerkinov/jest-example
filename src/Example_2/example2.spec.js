import React from "react";
import Posts from "./example2";

describe("Posts component", () => {
  it("should render Post component", () => {
    // const component = render(<Posts />);
    const component = shallow(<Posts />);
    expect(component).toMatchSnapshot();
  });
});
