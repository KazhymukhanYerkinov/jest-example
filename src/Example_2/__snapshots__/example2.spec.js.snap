// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Posts component should render Post component 1`] = `
<ul
  className="newsList"
>
  <Example1
    author="Yauhen"
    created_at="2020-05-03T23:36:09.816Z"
    key="1"
    num_comments={10}
    points={100}
    title="Jest & Enzyme"
    url="//test.url"
  />
  <Example1
    author="Stepan"
    created_at="2020-05-05T23:36:09.816Z"
    key="2"
    num_comments={8}
    points={10}
    title="TypeScript Basics"
    url="//test2121.url"
  />
</ul>
`;
